src/01-functions.js

```diff
@@ -93,7 +103,9 @@ function average() {}
  *  averageThree(10, 11, 19);
  *  //> 10 // 10 + 11 + 19 = 30; 30 / 3 = 10;
  */
-function averageThree() {}
+function averageThree(a,b,c) {
+  return (a+b+c)/3;
```

Comment:
You will most likely want to space out your variables both as function arguments and in their operations.  Example: we would want to write `averageThree(a,b,c)` => `averageThree(a, b, c)`. and `a+b+c)/3`=>` (a, b, c) / 3`